autoload/neomake/makers/ft/elixir.vim

```diff
@@ -1,12 +1,34 @@
 " vim: ts=4 sw=4 et
 
+" Credo error types.
+" F -> Refactoring opportunities
+" W -> Warnings
+" C -> Convention violation
+" D -> Software design suggestions
+" R -> Readability suggestions
+" Map structure {CredoError: NeomakeType, ...}
+let s:neomake_elixir_credo_type_map = {
+    \ 'F': 'W',
+    \ 'C': 'W',
+    \ 'D': 'I',
+    \ 'R': 'I'}
+
 function! neomake#makers#ft#elixir#PostprocessEnforceMaxBufferLine(entry) abort
     let buffer_lines = str2nr(line('$'))
     if (buffer_lines < a:entry.lnum)
         let a:entry.lnum = buffer_lines
     endif
 endfunction
 
+function! neomake#makers#ft#elixir#PostprocessCredo(entry) abort
+    let type = toupper(a:entry.type)
+    let type_map = extend(s:neomake_elixir_credo_type_map,
+                \ get(g:, 'neomake_elixir_credo_type_map', {}))
```

Comment:
well, now that we talk about that, i'm wondering if this config is actually needed. 
Perhaps mapping all the credo "errors" should be mapped to neomake warnings, without any configuration whatsoever. 
In the end, if you have something configured in your `.credo.exs` file, it's because you want  to enforce that rule.
In the beginning, I thought it might be useful, but if this is something that is not currently done in neomake, perhaps it is not the best case to introduce it